src/gui/ui_settings.c

```diff
@@ -442,16 +444,17 @@ static int settings_loop(int id, void *context, const input_data *input) {
       if (!left && !right) {
           break;
       }
-      char *resolutions[] = {"960x540", "960x544", "1280x720", "1920x1080"};
+      char *resolutions[] = {"960x544", "960x540", "1280x720", "1920x1080", "1280x540"};
```

Comment:
could you sort this resolutions?

Reply:
In which order? This is ordered by display ratio first then by vertical resolution.

Reply:
please change these to
960x540, 960x544, 1280x540, 1280x720, 1920x1080